package.json

```diff
@@ -36,7 +36,9 @@
     "snyk-wizard": "npm run snyk wizard",
     "start": "npm run version-check && better-npm-run --silent node bin/server.js | pino-mozlog",
     "start-func-test-server": "better-npm-run node bin/start-func-test-server.js",
-    "storybook": "better-npm-run storybook-build",
+    "storybook": "better-npm-run storybook",
+    "storybook-build": "better-npm-run storybook-build",
+    "storybook-deploy": "storybook-to-ghpages --script storybook-build",
```

Comment:
Looks like you can omi `--script` since the value is the default value: https://github.com/storybooks/storybook-deployer/blob/master/bin/storybook_to_ghpages#L24.

Reply:
hey @willdurand, right..I updated this based on [your comment](https://github.com/mozilla/addons-frontend/pull/6202#discussion_r217084368)

I had to pass in this option to use our custom named scripts (`storybook-build` vs their name of `build-storybook`). Otherwise we  need to use their name. let me know if that's what you think is better and I'll revert back to this:

![Alt text](https://monosnap.com/image/Hq1YJDeHOohPnbui8Uxm7jXaHqpPU4.png)

I personally like the scripts all starting with storybook too (I thought that's  what you were saying above) but I'll defer to you :)

Reply:
ah, right. yeah leave it as is with scripts starting with "storybook"